.changeset/odd-shirts-think.md

```diff
@@ -0,0 +1,5 @@
+---
+"vercel": patch
```

Comment:
```suggestion
"vercel": minor
```

I'd argue that it's a new feature and thus should be minor. Don't have a strong preference though, so feel free to ignore.